lib/rules/require-line-break-after-variable-assignment.js

```diff
@@ -61,6 +61,12 @@ module.exports.prototype = {
     check: function(file, errors) {
         var lastDeclaration;
         file.iterateNodesByType('VariableDeclaration', function(node) {
+            if (node.parentNode.type === 'ForStatement' ||
```

Comment:
No, no reason. If `node.parentNode.type` is empty — block will be skipped, if it contains wrong value — it will be also skipped, all is fine as is.